esx_service/vmdk_ops_test.py

```diff
@@ -972,9 +992,8 @@ def test_vmdkops_on_tenant_vm(self):
         # "result"" should have two volumes :tenant1_vol2@default_datastore, and  tenant1_vol3@datastore
         # these volumes are created at default_datastore, which is self.datastore_name
         self.assertEqual(2, len(result))
-        self.assertEqual("tenant1_vol2@"+self.datastore_name, result[0]['Name'])
-        self.assertEqual("tenant1_vol3@"+self.datastore_name, result[1]['Name'])
-
+        volume_names = ["tenant1_vol2@" + self.datastore_name, "tenant1_vol3@" + self.datastore_name]
```

Comment:
@shuklanirdesh82  Test 'test_vmdkops_on_tenant_vm' creates a tenant and then attaches volume tenant1_vol1_name and then detaches the volume. And then finally checks that 'result' should have two volumes :tenant1_vol2@default_datastore, and tenant1_vol3@datastore. That's why I didn't use this method here. I have used this method at instances where code checks for all the three volumes. Suggestions?